src/Pages/2 Using Blake/SiteTemplates.md

```diff
@@ -9,17 +9,15 @@ pageOrder: 4
 category: "Using Blake"
 ---
 
-:::info
-**Summary**
+:::summary
 Site templates provide complete starting points for your Blake projects. Use `blake new -t <template-name>` to scaffold a full site with layouts, styling, and example content tailored to specific use cases.
 :::
 
 ## Understanding Site Templates
 
 Site templates are different from [page templates](/pages/2%20using%20blake/page-templates) - they provide complete project scaffolding rather than individual page rendering.
 
-:::info
-**Why "Site Templates" Instead of "Themes"?**
+:::summary
```

Comment:
This change is not correct. Only the actual page summary should switch to these containers. Info containers are still valid elsewhere.